modules/serverless-logic/pages/core/timeouts-support.adoc

```diff
@@ -136,6 +213,11 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
 ==== Ephemeral
 Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
 
+====
```

Comment:
```suggestion
```

Reply:
Also appears in upstream: https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/core/timeouts-support.html#_job_service_persistence